README.md

```diff
@@ -78,10 +78,18 @@ they don't violate our quality standards. To avoid frustration, before
 sending us your pull request please run full Maven build:
 
 ```
-$ mvn clean install -Pqulice
+$ mvn clean install -Pqulice -PDockerITs
```

Comment:
@original-brownbear BTW, shouldn't this be the default? We could only provide possibility to skip those